guidelines/templates/resources/custom-resource.md

```diff
@@ -0,0 +1,36 @@
+---
+title: CRD name
+type: Custom Resource
+---
+
+> This document is a ready-to-use template for a Custom Resource (CR) document that provides the sample Custom Resource and description of its elements for a given CRD. For details, see the existing CR documents for the [Installer](https://github.com/kyma-project/kyma/blob/master/docs/kyma/docs/040-installation-custom-resource.md) and the [Api-gateway](https://github.com/kyma-project/kyma/blob/master/docs/api-gateway/docs/011-api-custom-resource.md).
+
+> **NOTE:** Blockquotes in this document provide instructions. Remove them from the final document.
+
+
+The {CRD full name with the domain} Custom Resource Definition (CRD) is a detailed description of the kind of data and the format used to {rest of the CRD description}. To get the up-to-date CRD and show the output in the yaml format, run this command:
+
+```
+kubectl get crd {CRD full name with the domain} -o yaml
+```
+
+## Sample Custom Resource
+
+This is a sample CR that {description of the CR functions}.
```

Comment:
Instead of this sentence, I think there should be a general explanation (block-quote format) telling the user what needs to go into this section and how to describe it. 

The section should contain a Custom Resource created using the CRD mentioned in the introductory section. 
The CR should be a valid one. If this is impossible, it should be as close to being valid as it is possible. 
Its functionality should be described, stressing any options used. See the examples below, the "options" are in bold.

> This is a sample CR that the API-Controller listens for to expose a service. **This example has the authentication section specified which makes the API Controller create an Istio Authentication Policy for this service.** 

You don't have to specify this section. If you don't, the service doesn't get an Istio Auth Policy. 

> This is a sample CR that controls the Kyma installer. **This example has the action label set to install**, which means that it triggers the installation of Kyma.

You could set the label to **uninstall** and get a different result when you apply the CR to the cluster.

Reply:
what you are suggesting is that templates:
- addresses that we want content provider to explain what example shows
- addresses option that content provider can add more then one example
- and that above is explained in a quote

I agree with above